lib/content-types/only.js

```diff
@@ -16,16 +16,32 @@ const only = (type, config) => {
         return attr;
       }
 
-      // Merge configuration for the inputs and the configuration
-      attr.inputs = merge(attr.inputs, config[attr.id]);
-
-      // Override the validation, type, and ID for all inputs that matter
-      inputs.forEach(input => {
-        attr.inputs[input].validation = attribute.inputs[input].validation;
-        attr.inputs[input].type = attribute.inputs[input].type;
-        attr.inputs[input].id = attribute.inputs[input].id;
-        attr.inputs[input].name = attribute.inputs[input].name;
-      });
+      // Flattens multiple instances
+      if (Array.isArray(config[attr.id])) {
```

Comment:
Inputs would be an array for the multiple instances. 
The value of select and checkbox will be stored as string and array respectively